services/streammanager/kafka/kafkamanager.go

```diff
@@ -307,7 +311,7 @@ func NewTLSConfig(caCertFile string) *tls.Config {
 	tlsConfig.RootCAs = caCertPool
 
 	//tlsConfig.BuildNameToCertificate()
-	tlsConfig.InsecureSkipVerify = true
+	tlsConfig.InsecureSkipVerify = false
```

Comment:
I remember having issues with some existing clients with expired certificates.
I believe this should be configurable even if the option is hidden from the UI. So that we have the option of applying a quick fix for clients with expired certificates and similar SSL issues. 

Especially since we don't have a way of notifying them, that their keys are expired.

Reply:
For quick fix , We could disable SSL which would mean it would not even use this function.This flag essentially skips verifying the certificate so I guess turning off the flag in the mean time would do no harm

Reply:
Not sure if it would be possible to connect to an SSL endpoint with SSL disabled on the client-side.

Reply:
Yeah makes sense